pkg/virt-controller/watch/application.go

```diff
@@ -69,7 +76,21 @@ func Execute() {
 
 	app.restClient = app.clientSet.RestClient()
 
-	restful.Add(rest.WebService)
+	readinessFunc := func(_ *restful.Request, response *restful.Response) {
+		res := map[string]interface{}{}
+		if !app.isLeader {
+			res["apiserver"] = map[string]interface{}{"leader": "failed", "error": "current pod is not leader"}
```

Comment:
Error does not seem appropriate. Could  we use: `"leader": "true"` and `"leader": "false"` instead?